spec/source/conventions.html

```diff
@@ -0,0 +1,107 @@
+<emu-clause id="conventions">
+  <h1>Notational Conventions</h1>
+
+  <p>
+    This standard uses a subset of the notational conventions of the ECMAScript 2015 Language Specification (ECMA-262 6th Edition), as ES2015:
+  </p>
+
+  <ul>
+    <li>Object Internal Methods and Internal Slots, as described in ES2015, 6.1.7.2.</li>
+    <li>Algorithm conventions, including the use of abstract operations, as described in ES2015, 7.1, 7.2, 7.3.</li>
+    <li>Internal Slots, as described in ES2015, 9.1.</li>
+    <li>The List and Record Specification Type, as described in ES2015, 6.2.1.</li>
+  </ul>
+
+  <emu-note>
+    <p>
+      As described in the ECMAScript Language Specification, algorithms are used to precisely specify the required semantics of ECMAScript constructs, but are not intended to imply the use of any specific implementation technique. Internal slots are used to define the semantics of object values, but are not part of the API. They are defined purely for expository purposes. An implementation of the API must behave as if it produced and operated upon internal slots in the manner described here.
+    </p>
+  </emu-note>
+
+  <p>
+    As an extension to the Record Specification Type, the notation "[[name]]" denotes a field whose name is given by the variable name, which must have a String value. For example, if a variable s has the value "a", then [[s]] denotes the field [[a]].
+  </p>
+
+  <p>
+    For ECMAScript objects, this standard may use variable-named internal slots: The notation "[[name]]" denotes an internal slot whose name is given by the variable name, which must have a String value. For example, if a variable s has the value "a", then [[s]] denotes the [[a]] internal slot.
```

Comment:
>  I rename them to just [[s]], I think going forward we might want to use that notation, not sure.

No, that's wrong. The `[[<s>]]` notation has a specific meaning that will be completely broken if these are just changed to `[[s]]`. 

![image](https://cloud.githubusercontent.com/assets/27985/8835257/456bf4de-3087-11e5-87f3-45e72d961fd3.png)

Is it breaking because the literal `[[<s>]]` is interpreted as though `<s>` is an HTML tag? If that's the case, then just write it out like `[[&lt;s&gt;]]` for these normative explanations. Then in the algorithms, escape them: `[[\<s\>]]`